.circleci/create_circleci_config.py

```diff
@@ -472,6 +472,8 @@ def job_name(self):
         "pip install -U --upgrade-strategy eager 'git+https://github.com/facebookresearch/detectron2.git'",
         "sudo apt install tesseract-ocr",
         "pip install -U --upgrade-strategy eager pytesseract",
+        'pip install -U --upgrade-strategy eager "Pillow<10.0.0"',
```

Comment:
there were security issues with previous pillow versions + this will install this particular version for all the model, not sure we should do it.